packages/tools/pie-icons-webc/build.js

```diff
@@ -38,6 +38,9 @@ const componentSelector = '${kebabCase(name)}';
 export class ${name} extends LitElement implements IconProps {
     // The following styles make sure that the icon will be sized correctly
     static styles = css\`
+        :host {
+            display: inline-block;
+        }
         :host svg {
```

Comment:
Could you please add a comment here to clarify the reasoning for this override vars, something like this:
// Variables can be used in parent components to make sure that the icon stays the correct size for cases where the icon is added by consumers via a slot, for example in IconButton 
// Might be changed later to assigning a size prop to an icon slot (POC: https://github.com/justeattakeaway/pie/pull/1128)